apps/envited-x-data-space/common/database/queries/users.ts

```diff
@@ -48,23 +51,69 @@ export const getUserById = (db: DatabaseConnection) => async (id: string) =>
           },
         },
       },
+      urnGlobalIdentifier: true,
+      addressGlobalIdentifier: true,
     },
   })
 
 export const getUserRolesById = (db: DatabaseConnection) => async (id: string) =>
   db.select().from(usersToRoles).where(eq(usersToRoles.userId, id))
 
-export const getUserWithProfileById = (db: DatabaseConnection) => async (id: string) =>
-  db.select().from(user).where(eq(user.id, id)).leftJoin(profile, eq(user.name, profile.name))
-
-export const getUserByName = (db: DatabaseConnection) => async (name: string) =>
-  db.select().from(user).where(eq(user.name, name))
+export const getUserRolesByDid =
```

Comment:
Is this name correct, normally we use ByDid when you throw in a did but that's not the case anymore

Reply:
It is the did that goes in, but split into its respective parts